src/styles/globals.css

```diff
@@ -10,6 +10,20 @@ html {
   background-color: #101010;
 }
 
+.if-mobile {
```

Comment:
bro for responsive please dont use low-level css media screen, tailwind already provided us that functionality: https://tailwindcss.com/docs/responsive-design

Reply:
But if i use tailwind, i can't do it.

Reply:
tailwind responstive is mobile first bro. so anything that is not mobile you can prefix sth like  hidden md:block effectively this whole thing will mean on mobile hide it and md(ipad) and above show it.